nornir_netbox/plugins/inventory/netbox.py

```diff
@@ -308,22 +310,32 @@ def load(self) -> Inventory:
         groups = Groups()
         defaults = Defaults()
 
-        if self.defaults_file.exists():
-            with self.defaults_file.open("r") as f:
-                defaults_dict = yml.load(f) or {}
-            defaults = _get_defaults(defaults_dict)
-        else:
-            defaults = Defaults()
+        try:
+            if self.defaults_file.exists():
+                with self.defaults_file.open("r") as f:
+                    defaults_dict = yml.load(f) or {}
+                defaults = _get_defaults(defaults_dict)
+        except PermissionError as e:
+            if not self.ignore_file_errors:
+                raise PermissionError(e)
```

Comment:
`raise` should be sufficient here to re-raise the current handled exception
would add a logging message (error level)